### PR TITLE
Add sentry support

### LINKED ISSUE #50: Add Sentry support for UI & Main Process

### PR DESCRIPTION
**Pull request by ianaya89**
Resolves #50

### CODE REVIEW

File: public/electron.js [hunk 1]
```diff
@@ -42,7 +59,7 @@ createWindow()
 const { initMainWorker } = require(path.join(__dirname, './main/mainWorker.js'))
 
 app.on('ready', function() {
-  logger.info('App ready, initlilizing...')
+  logger.info('App ready, initializing 🚀')
```
Comment (gabmontes) [inline public/electron.js]: Nice rocket but prefer plain text in logs. Don't want to shut remote loggers down with Unicode chars. Please remove it.

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+
 import theme from './theme'
-import React from 'react'
+import config from './config'
 import App from './components/App'
+import createStore from './createStore'
+
+if (config.SENTRY_DSN) {
+  Raven.config(
+    'https://d99caae5ca1b495584e5aa3187966f30@sentry.io/277183'
```
Comment (gabmontes) [inline src/index.js]: This should be taken from `config.SENTRY_DSN`.

**Comment by gabmontes** [comment]
@ianaya89 did you cancel this PR?

**Review by eafelix — APPROVED** [review]
💯

File: public/electron.js (hunk 1 above)
Comment (eafelix) [reply public/electron.js]: where could it represent a problem?

**Review by eafelix — CHANGES_REQUESTED** [review]

**Review by eafelix — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]

**Pull request merged**